app/src/main/res/layout/fragment_view_health_prof_page.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please rewrite as labels and value to present information. Also missing other values for health professional. Where does specialty come from? There's no specialty input in the create health prof page.

Reply:
done change